content/links.yaml

```diff
@@ -761,3 +761,31 @@
       - scipy
       - numpy
       - matplotlib
+- title: Pangeo Gallery
+  description: A gallery of different Pangeo use cases
+  url: http://gallery.pangeo.io
+  thumbnail: https://github.com/pangeo-data/pangeo/raw/master/docs/_static/pangeo_simple_logo.png
+  authors:
+    - name: Pangeo Community
+      affiliation: Pangeo
+      affiliation_url: http://pangeo.io/
+      email: ''
+  tags:
+    packages:
+      - jupyter
+      - xarray
+      - dask
+      - matplotlib
+      - intake
+      - xgcm
+      - holoviews
+      - pandas
+      - numpy
+    formats:
+      - gallery
+      - tutorial
+    domains:
+      - glaciology
+      - remote-sensing
+      - physical-oceanography
```

Comment:
I'm not a huge fan of the dashes in the domains as they appear in the pull-down menu in the links gallery (e.g. `Physical-Oceanography`). Do they need to be here in the yaml to avoid breaking something?

Reply:
Also in this PR the Domains menu contains both "Remote Sensing" and "Remote-Sensing"

Reply:
No they don't need to be there, they're just in the address. Let me look into this.

Reply:
Easy enough fix. See the new PR https://github.com/ProjectPythia/.github/pull/7

Reply:
Great! Better now.